### ISSUE TITLE
Feature request: in the ui select minimum and maximum of a metric

### ISSUE THREAD

**Issue by stablum** [mlflow/mlflow#1422]
The current web ui of mlflow, in the runs' table summary shows the last value of the metrics.
It would be nice to see the minimum and the maximum (of the error for example) as they might be more relevant. Especially to detect the "bouncing" overfitting curve on the validation set.

**Comment by smurching** [comment]
Hi @stablum, thanks for bringing this up. Would it suffice to view this info for a single run, or is it important to be able to view it across runs?

**Comment by stablum** [comment]
It's important to view it across runs and to sort on it. This will help finding the parameter configurations that were leading to better optima.

**Comment by SebastianRiechert** [comment]
any progress on this? Just realized this isn't possible yet
